src/main/java/uk/gov/hmcts/reform/orgrolemapping/controller/WelcomeController.java

```diff
@@ -51,48 +25,12 @@ public RedirectView swaggerRedirect() {
     public String welcome() {
         return "Welcome to Organisation Role Mapping Service";
     }
-    //This is just a test API
-
-    @PostMapping(
-            path = "/am/role-mapping/staff/users",
-            produces = V1.MediaType.MAP_ASSIGNMENTS,
-            consumes = {"application/json"}
-    )
-    @ResponseStatus(code = HttpStatus.OK)
-    @Operation(summary = "creates multiple role assignments based upon user profile mapping rules",
-            security =
-            {
-                @SecurityRequirement(name = AUTHORIZATION),
-                @SecurityRequirement(name = SERVICE_AUTHORIZATION)
-            })
-    @ApiResponse(
-            responseCode = "200",
-            description = "OK",
-            content = @Content(schema = @Schema(implementation = Object.class))
-    )
-    @ApiResponse(
-            responseCode = "400",
-            description = V1.Error.INVALID_REQUEST,
-            content = @Content()
-    )
-    public ResponseEntity<Object> createOrgMapping(@RequestBody UserRequest userRequest,
-                                                   @RequestHeader(value = "userType")
-                                                           UserType userType) {
-        var startTime = System.currentTimeMillis();
-        log.debug("createOrgMapping");
-        log.info("Process has been Started for the userIds {}", userRequest.getUserIds());
-        ResponseEntity<Object> response = bulkAssignmentOrchestrator.createBulkAssignmentsRequest(userRequest,
-                userType);
-        log.debug("Execution time of createOrgMapping() : {} ms",
-                (Math.subtractExact(System.currentTimeMillis(), startTime)));
-        return ResponseEntity.status(response.getStatusCode()).body(response.getBody());
-    }
-
 
     //This method needed for the functional tests, so that RAS gets enough time to create records.
     @PostMapping(value = "/sleep")
     public ResponseEntity<String> waitFor(String duration) {
```

Comment:
the String `duration` is a unused variable. should this be removed?

Reply:
This would be better addressed in [AM-2857](https://tools.hmcts.net/jira/browse/AM-2857) _"Clean up Welcome Controller (ORM)"_, see #1498.